src/plugins/data/public/plugin.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { PluginInitializerContext, CoreSetup, CoreStart } from '../../../core/public';
+import { setup } from './interpreter/setup';
+
+export class Plugin {
+  constructor(initializerContext: PluginInitializerContext) {}
+
+  public setup(core: CoreSetup) {
+    const interpreter = setup();
+    return {
+      interpreter,
+    };
+  }
+
+  public start(core: CoreStart) {
+    return {};
+  }
```

Comment:
I think core is going to expect a `stop` method to exist too, even if it does nothing, right? I thought that was required, but could be wrong.

Reply:
Yes, I'll add `stop`, we will need it for other services anyways. I don't know  if it is required, but in their demo `testbed` plugin they don't have `stop` life-cycle.

Reply:
Added. `stop()` was optional in `Plugin` interface, but it looks like core calls it without checking if it exists, fixed the interface.